src/app/qgscrashhandler.cpp

```diff
@@ -59,7 +59,12 @@ LONG WINAPI QgsCrashHandler::handle( LPEXCEPTION_POINTERS exception )
   arguments = QCoreApplication::arguments();
   // TODO In future this needs to be moved out into a "session state" file because we can't trust this is valid in
   // a crash.
-  arguments << QgsProject::instance()->fileName();
+  QString projectFile = QgsProject::instance()->fileName();
+  if ( !projectFile.isEmpty() )
+    // quote project file path to avoid issues if it has spaces
+    arguments << '"' << projectFile << '"';
```

Comment:
```suggestion
    arguments << QStringLiteral( "\"%1\"" ).arg( QgsProject::instance()->fileName() );

otherwise it's creating multiple entries in the list instead of a single string...

Reply:
Oh, my bad. Fixed.